components/00-base/typography/type-faces.yml

```diff
@@ -0,0 +1,34 @@
+families:
```

Comment:
More of a question/consideration than feedback to implement : Should `line-height` also be considered here? I see that it's being set here: https://github.com/yalesites-org/tokens/blob/main/tokens/typography/font.yml#L82 but no reference within this PR.

Reply:
Not on this display. It's just there to show the different families that can be used. The "styles" display includes line-height and letter-spacing, and all the things.